testing/scripts/seldon_e2e_utils.py

```diff
@@ -42,6 +43,27 @@ def wait_for_shutdown(deploymentName, namespace):
         ret = run(f"kubectl get -n {namespace} deploy/{deploymentName}", shell=True)
 
 
+def get_deployment_names(seldonDeploymentName, namespace, attempts=20, sleep=5):
```

Comment:
Let's stick with underscore naming conventions

Reply:
Like `seldonDeploymentName` -> `seldon_deployment_name`? 
I was following convention present already in the file.

Shall I apply underscore convention throughout the file?

Reply:
Changed everywhere in the file.

Reply:
@axsaucedo I already applied change in favour of underscores...